File: didsdk/ios/Classes/SDKErrors.swift
internal class SDKErrors {
    static let defaultError = FlutterError.init(code: "99", message: "Unknown library error", details: nil)
    static let errorNotArguments = FlutterError.init(code: "98", message: "Missing arguments", details: nil)
    static let errorChannelMethod = FlutterError.init(code: "99", message: "Unknown method in this channel", details: nil)
    static let ungrantedPushPermission = FlutterError.init(code: "85", message: "Ungranted Push Notifications permission", details: nil)
    static let invalidAPNS = FlutterError.init(code: "86", message: "Invalid APNS environment", details: nil)
}

File: didsdk/android/src/main/kotlin/com/appgate/didsdk/constants/SDKErrors.kt
package com.appgate.didsdk.constants

enum class SDKErrors(val code: String, val message: String, val details: String?) {
    DEFAULT_ERROR("99", "Unknown library error", null),
    ERROR_NOT_ARGUMENTS("99", "Missing arguments", null),
    ERROR_CHANNEL_METHOD("99", "Unknown method in this channel", null),
    INVALID_CHALLENGE_LENGTH("88", "onInvalidChallengeLength", null),
    PARAMETER_MISSING("98", "ErrorParameterMissing", null),
}